project/Build.scala

```diff
@@ -35,8 +35,9 @@ object Build extends Build {
       "org.slf4j" % "log4j-over-slf4j" % Slf4jVersion % "test",
       "org.mockito" % "mockito-all" % MockitoVersion % "test",
       "org.scalatest" %% "scalatest" % ScalatestVersion % "test",
-      "org.codehaus.groovy" % "groovy" % GroovyVersion % "test"
-
+      "org.codehaus.groovy" % "groovy" % GroovyVersion % "test",
+      "org.codehaus.groovy" % "groovy" % GroovyVersion % "test",
+      "com.vividsolutions" % "jts" % "1.13" % "test"
```

Comment:
I don't see any new imports which should come from this new dependency.
Did I miss something?
Maybe it is an `optional` dependency in ElasticSearch ?!